Sources/SnowManLogic/Game.swift

```diff
@@ -0,0 +1,15 @@
+import Dictionary
+
+public class Game {
+	let dictionary: DictionaryManager!
```

Comment:
the `!` is not necessary, since you're setting it in from the initializer and it's a constant.